cmd/channel/distributed/receiver/main.go

```diff
@@ -134,8 +135,13 @@ func main() {
 	// Create A New Stats StatsReporter
 	statsReporter := metrics.NewStatsReporter(logger)
 
+	// Create A Watcher On The Configuration Settings ConfigMap & Dynamically Update Configuration
+	// Since this is designed to be called by the main() function, the default KNative package behavior here
+	// is a fatal exit if the watch cannot be set up.
+	cmw := sharedmain.SetupConfigMapWatchOrDie(ctx, logger.Sugar())
```

Comment:
You forgot to start the config map watcher, right? Or does this method start it?